Adherence to Agent Friendly Docs Specification (#3576)
apache/tinkerpop: docs/src/recipes/element-existence.asciidoc

specific language governing permissions and limitations
under the License.
////

> [llms-summary="Recipe: Checking for whether or not a graph element is present in the graph is simple:"]

Reviewer: ```suggestion
[llms-summary="Recipe: Checking whether a graph element (a vertex or edge) is already present in the graph."]
```
